src/onboarding/registration/NameAndPicture.test.tsx

```diff
@@ -183,32 +192,42 @@ describe('NameAndPictureScreen', () => {
     expect(getByText('nameAndPicGuideCopyContent')).toBeTruthy()
   })
   it('does not render skip button when configured so', () => {
-    // TODO replace route param with mock Statsig flag
+    ExperimentParams[StatsigLayers.NAME_AND_PICTURE_SCREEN].showSkipButton.defaultValue = false
```

Comment:
tests shouldn't mutate constants from application code. if you want to do this, find a way to mock ExperimentParams and mess with that, instead. or refactor to use dependency injection

Reply:
tbh I don't think it's necessary to test fictitious variations of the default value though

Reply:
Ok makes sense. Also, I'll update these tests in a separate PR.